### PR TITLE
feat auto import plugin

### LINKED ISSUE #304: ✨ [Feature]:  Please support unplugin-vue-components
### What problem does this feature solve

不积极加入vue开发相关生态，谁知道有个什么 opentiny，还要自己写按需加载的解析器，对开发者太不友好了。

### What does the proposed API look like

添加 [unplugin-vue-components](https://github.com/antfu/unplugin-vue-components) 组件

### PR DESCRIPTION
**Pull request by mengqiuleo**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #304

## What is the new behavior?
close #304 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by mengqiuleo** [comment]
讨论：https://github.com/opentiny/tiny-vue/discussions/1396

效果：

File: internals/unplugin-tiny-vue/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "unplugin-tiny-vue",
```
Comment (kagol) [inline internals/unplugin-tiny-vue/package.json]:
```suggestion
  "name": "@opentiny/unplugin-tiny-vue",
```

File: internals/unplugin-tiny-vue/package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
```
Comment (kagol) [inline internals/unplugin-tiny-vue/package.json]: 我看产物在dist/index.js，这样写发包时会不会有问题呢？
我应该在根目录执行：npm publish 进行发包，还是在 dist 目录执行呢？

Comment (mengqiuleo) [reply internals/unplugin-tiny-vue/package.json]: 修复了，会和 已有插件 保持一致。

**Comment by mengqiuleo** [comment]
我 git push -f 之后，Vercel 部署还是失败？

**Comment by kagol** [comment]
> 我 git push -f 之后，Vercel 部署还是失败？

不会呀，我看已经成功了，可以正常预览。
image

**Review by zzcr — APPROVED** [review]

**Pull request merged**